From 94472e1cc130a5fe4aeca2373d21f9eb61d33f5f Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 27 Mar 2014 11:38:34 +0000
Subject: [PATCH 0454/1000] MANTIS 29186 Mantis 29186: GPPDev legt
 pectabConversion Verzeichnis nicht an Use the "try_create" option in
 "mSysProcessListW32.cpp", which in turn will affect mStarter operation

git-svn-id: svn://localhost/SelfServiceCommon/trunk@661 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Systools/src/mSysProcessListW32.cpp             | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/cpp/Systools/src/mSysProcessListW32.cpp b/SelfServiceCommon/Massai/cpp/Systools/src/mSysProcessListW32.cpp
index 34587557..47833e3d 100644
--- a/SelfServiceCommon/Massai/cpp/Systools/src/mSysProcessListW32.cpp
+++ b/SelfServiceCommon/Massai/cpp/Systools/src/mSysProcessListW32.cpp
@@ -714,7 +714,8 @@ namespace
     factory.setReuseConsole();
     if(m_properties.workingDir.size() > 0)
     {
-      factory.setWorkDirectory(m_properties.workingDir.c_str());
+      // Select the working directory with the "try_create" option set
+      factory.setWorkDirectory(m_properties.workingDir.c_str(),true);
     }
     switch(m_properties.show)
     {
-- 
2.41.0.windows.1

